src/features/home/routes/home.routes.ts

```diff
@@ -8,6 +8,9 @@ const HomeScreen = React.lazy(() => import("../screens/HomeScreen/HomeScreen"));
 const HOME_SCREEN: RouteItemDef = {
   path: HomePathsEnum.HOME,
   component: HomeScreen,
+  icon: "dashboard",
```

Comment:
should it maybe instead of a string, be the actual svg icon fra material ui?

Reply:
I used the material icon svg component but there's one thing that I couldn't figure out which is the Type of it. I tried ReactElement but didn't work.